staramr/blast/JobHandler.py

```diff
@@ -93,8 +94,24 @@ def run_blasts(self, files) -> None:
         db_files = self._make_db_from_input_files(self._input_genomes_tmp_dir, files)
         logger.debug("Done making blast databases for input files")
 
-        logger.info("Scheduling MLST for input files")
-        self._schedule_mlst(db_files)
+        future_mlst_db = [] # type: list
+
+        if self._thread_pool_executor is not None:
+          logger.info("Scheduling MLST for input files")
+          future_mlst_db.append(self._thread_pool_executor.submit(self._schedule_mlst, db_files))
```

Comment:
If you wrap this in a `for` loop iterating over each file in `db_files`, it will schedule them all in parallel. Right now, it is only scheduling one single `mlst` job with all the files.

Reply:
Thanks for the suggestion! I'll include this in my next commit